### ISSUE TITLE
Attack Monitor fails to run due to the FORCE_EMPTY_DIR dummy file

### ISSUE THREAD

**Issue by ewilded** [yarox24/attack_monitor#1]
First, I installed the tool like this (correct me if some steps were not needed, basically I just wanted to make sure I have all the capabilities).

First I ran `python installer.py sysmon`  to get proper sysmon config (was already there in my case):

![image](https://user-images.githubusercontent.com/1158719/56759701-ddc76400-6799-11e9-8a1d-25dacebe5f7e.png)

Then I ran `python installer.py install auditpol` (`python installer.py auditpol` was returning an error, just wanted to be sure I have it installed with the extended audit).

Then I installed dependencies (by the way, I installed in the Endpoint Protection Mode, so pyshark and the reportlab should not really be required, should they?).

I also installed the pre-defined exceptions:

![image](https://user-images.githubusercontent.com/1158719/56760113-ca68c880-679a-11e9-8483-13ecea73179d.png)

Then I tried to run it and got this issue:
![image](https://user-images.githubusercontent.com/1158719/56759942-63e3aa80-679a-11e9-9c46-43f375a3f71b.png)

So I dug a bit, added one print to the code to get some verbose output:

![image](https://user-images.githubusercontent.com/1158719/56759983-79f16b00-679a-11e9-842a-0a9b6d32aa10.png)

Tried to run again and got this:

![image](https://user-images.githubusercontent.com/1158719/56760007-85dd2d00-679a-11e9-8809-5823b1e2cc08.png)

So I figured out the culprit was the dummy FORCE_EMPTY_DIR file, which is not an actual JSON file:

![image](https://user-images.githubusercontent.com/1158719/56760041-968da300-679a-11e9-9b5b-7eead4a1a535.png)

So I removed it and ran the tool again, this time successfully :)

**SOLUTION:**
Remove that file while installing/avoid installing it in the first place.

**Comment by yarox24** [comment]
Yes, I need to fix this. As I uploaded the project there was a problem with empty directories. Github doesn't accept upload of empty directories, so I added dummy files to force their creation. But it broke exceptions loading.
